README.md

```diff
@@ -1,7 +1,9 @@
 # Datadog CDK Constructs
 
-[![NPM](https://img.shields.io/npm/v/datadog-cdk-constructs?color=blue)](https://www.npmjs.com/package/datadog-cdk-constructs)
-[![PyPI](https://img.shields.io/pypi/v/datadog-cdk-constructs?color=blue)](https://pypi.org/project/datadog-cdk-constructs/)
+[![NPM](https://img.shields.io/npm/v/datadog-cdk-constructs?color=blue&label=npm+cdk+v1)](https://www.npmjs.com/package/datadog-cdk-constructs)
+[![NPM](https://img.shields.io/npm/v/datadog-cdk-constructs-v2?color=blue&label=npm+cdk+v2)](https://www.npmjs.com/package/datadog-cdk-constructs-v2)
```

Comment:
I clicked on this cdk v2 npm badge and the npm page doesn't have a readme. Did something happen during publishing that the readme didn't get transferred to npm? From what I remember when I used to publish the cdk to npm it would take the repositories readme with it

Reply:
color coordinating the npm vs pypi badges might look nice too

Reply:
same with the pypi cdkv2 page https://pypi.org/project/datadog-cdk-constructs-v2/. The readme isn't there

Reply:
unfortunately publishing to npm and pypi pulls the readme from the root of the directory, rather than the root of this repo. This will effect future published versions of v1 as well. 

It might be a good idea to soft-link this readme into the v1 and v2 folders, replacing the placeholder readme files that exist in the two folders currently.